pkg/sql/row/expr_walker.go

```diff
@@ -146,7 +146,6 @@ func makeBuiltinOverride(
 // SequenceMetadata contains information used when processing columns with
```

Comment:
This commit has changes in this file which are the result of a tack I took that led nowhere. I left them in because why not.

Reply:
Ack.